frontend/src/providers/Dashboard/Dashboard.tsx

```diff
@@ -52,7 +52,7 @@ const DashboardContext = createContext<IDashboardContext>({
 	layouts: [],
 	panelMap: {},
 	setPanelMap: () => {},
-	listSortOrder: { columnKey: null, order: null, pagination: '1' },
+	listSortOrder: { columnKey: 'createdAt', order: 'descend', pagination: '1' },
```

Comment:
As @ankitnayan mentioned, I think the most recent updated entity should be on top.

Reply:
done